.github/workflows/test.yml

```diff
@@ -35,6 +35,12 @@ jobs:
       run: |
         pip install -r requirements.txt
         pip install -r dev-requirements.txt
+        pip install --upgrade pytest-rerunfailures
+        pip install -e 'git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-scheming.git@release-3.0.0#egg=ckanext-scheming[requirements]'
+        pip install -e 'git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-harvest.git@v1.5.6#egg=ckanext-harvest[requirements]'
+        pip install -e 'git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-dcat.git@v1.5.1#egg=ckanext-dcat[requirements]'
```

Comment:
`[requirements]` does not work for `ckanext-dcat`, you need to install `requirements.txt` directly from the local environment, after installing the plugin.

Reply:
corrected, I also replaced references to actual repositories rather than GDI forks